manifest.json

```diff
@@ -0,0 +1,51 @@
+{
+  "dxt_version": "0.1",
+  "name": "@microsoft/clarity-mcp-server",
+  "version": "1.0.5",
+  "description": "MCP Server for Microsoft Clarity based on data export API",
+  "long_description": "MCP Server for Microsoft Clarity based on data export API",
+  "author": {
+    "name": "Microsoft",
+     "url": "https://clarity.microsoft.com"
+  },
+  "icon": "icon.png",
+  "screenshots": [
+    "icon.png"
+  ],
+  "server": {
+    "type": "node",
+    "entry_point": "dist/index.js",
+    "mcp_config": {
+      "command": "node",
+      "args": [
+        "${__dirname}/dist/index.js"
+      ],
+      "env": {
+        "CLARITY_API_TOKEN": "${user_config.CLARITY_API_TOKEN}"
+      }
+    }
+  },
+  "tools_generated": true,
+  "keywords": ["api", "analytics", "productivity"],
```

Comment:
Using the following as keywords: [
    "clarity",
    "behavior-analytics-tools",
    "heatmaps",
    "session-recordings",
    "insights",
    "analytics",
    "mcp",
    "model-context-protocol",
    "ai",
    "agents"
]